src/less/techne-patterns/_product-thumbnail.less

```diff
@@ -1,28 +1,112 @@
-//         </div>
-//     </div>
-//     <div class="product-details">
-//         <div class="product-name">
-//             <a>Product Name</a>
-//         </div>
-//         <div class="product-code">
-//             000123456789000
-//         </div>
-//         <div class="product-price">
-//             <s>47.48</s>
-//             <span class="text-danger">42.48</span>
-//             <span>USD</span>
-//         </div>
-//         <div class="label label-success">Published</div>
-//     </div>
-// </div>
-//
-// Pattern 15.1
+/*
+Markup:
+ <div class="product-thumbnail">
+     <div class="product-image">
+         <div class="img-wrapper">
+         <span class="img-spacer"></span>
+         <img src="../images/product-thumbnail.png" class="img img-responsive full-width">
+         </div>
+     </div>
+     <div class="product-details">
+         <div class="product-name">
+             <a>Product Name</a>
+         </div>
+         <div class="product-code">
+             000123456789000
+         </div>
+         <div class="product-price ">
+             <span class="product-price-element">
+                <span class="product-price-amount">40.20</span>
+                <span class="product-price-currency">USD</span>
+             </span>
+         </div>
+         <div class="label label-success">Active</div>
+     </div>
+ </div>
+
+Pattern 15.1
+*/
+
+/*
+Display discounted pices
+
+The Product Tumbnail can also display disconted prices. The original price will be displayed besides the discounted price. Both price have different styles to be easily recognizable.
+
+Markup:
+ <div class="product-thumbnail">
+     <div class="product-image">
+         <div class="img-wrapper">
+         <span class="img-spacer"></span>
+         <img src="../images/product-thumbnail.png" class="img img-responsive full-width">
+         </div>
+     </div>
+     <div class="product-details">
+         <div class="product-name">
+             <a>Product Name</a>
+         </div>
+         <div class="product-code">
+             000123456789000
+         </div>
+         <div class="product-price ">
+             <span class="product-price-element product-price-original">
+                <span class="product-price-amount">47.48</span>
+             </span>
+             <span class="product-price-separator">&nbsp;</span>
+             <span class="product-price-element product-price-discounted">
+                <span class="product-price-amount">40.20</span>
+                <span class="product-price-currency">USD</span>
+             </span>
+         </div>
+         <div class="label label-success">Active</div>
+     </div>
+ </div>
+
+Pattern 15.2
+*/
+
+/*
+Display a price range
+
+The Product Thumbnail Pattern can also display price ranges. By adding the class .product-price-range to the element with the class .product-price-element the price ranges will be displayed one bellow the other.
```

Comment:
Display a price range
 
The Product Thumbnail Pattern can also display price ranges. By adding the class `product-price-range` to the element with the class `product-price-element` the price ranges will be displayed one below the other.